ocs_ci/ocs/cluster.py

```diff
@@ -586,6 +693,230 @@ def get_throughput_percentage(self):
         logging.info(f"The throughput percentage of the cluster is {throughput_percentage}%")
         return throughput_percentage
 
+    def osd_health_check(self, count):
+        """
+        Osd health check based on pod count
+
+        Args:
+            count (int): Expected number of Osd pods
+
+        Raises:
+            OsdCountException: if osd pod count doesn't match
```

Comment:
add an empty line below. Please do in all functions

Reply:
Done